chain/test/Test/Pos/Chain/Block/Arbitrary.hs

```diff
@@ -465,4 +477,13 @@ genHeaderAndParams pm = do
 -- type, so it is used here and at most 3 blocks are taken from the generated
 -- list.
 instance Arbitrary HeaderAndParams where
-    arbitrary = arbitrary >>= genHeaderAndParams
+    arbitrary = do
+        pm <- arbitrary
+        era <- arbitrary
+        genHeaderAndParams pm era
+
+-- TODO mhueschen | decide: should we generate both?
+genConsensusEra :: Gen ConsensusEra
+genConsensusEra = do
+    obftStrictness <- arbitrary
+    elements [Original, OBFT obftStrictness]
```

Comment:
`genConsensusEra` doesn't seem to be used anywhere in the codebase - can we drop it?

Reply:
Yeah, will drop it.